tools/firtool/firtool.cpp

```diff
@@ -74,6 +74,10 @@ static cl::opt<bool> inliner("inline",
 static cl::opt<bool> lowerToHW("lower-to-hw",
                                cl::desc("run the lower-to-hw pass"));
 
+static cl::opt<bool> enableAnnotationWarning("emit-annotation-warning",
+                             cl::desc("Run the FIRRTL module inliner"),
+                             cl::init(false));
```

Comment:
The first line is bikeshedding. The second may be this getting copied from earlier. 😄 

```suggestion
static cl::opt<bool> enableAnnotationWarning("warn-on-unprocessed-annotations",
                             cl::desc("Warn about annotations that were not removed by lower-to-hw"),
                             cl::init(false));
```

Reply:
Updated the command line option for both firtool and circt-opt